viz-lib/src/visualizations/chart/Editor/GeneralSettings.jsx

```diff
@@ -290,6 +292,41 @@ export default function GeneralSettings({ options, data, onOptionsChange }) {
           </Select>
         </Section>
       )}
+
+      {includes(["heatmap"], options.globalSeriesType) && (
+        <React.Fragment>
+          <Section>
+            <InputNumber
+              label="Brick Spacing"
+              data-test="Chart.HeatmapSpacing"
+              defaultValue={options.heatmapSpacing}
+              onChange={value => onOptionsChange({ heatmapSpacing: toNumber(value) })}
+            />
+          </Section>
+          <Section>
+            <Grid.Row gutter={15} type="flex" align="middle">
+              <Grid.Col span={12}>
+                <InputNumber
+                  label="zMin Value"
```

Comment:
I particularly find zMin and zMax names that can sound unfamiliar to users. @deecay what do you think? Minimum and _maximum Z-values_ can be better, but also may be verbose.

Reply:
Agree that these sound unfamiliar. Since Y axis settings has "Min Value/Max Value", how about "Color Column Min Value" and "Color Column Max Value"? More verbose than your suggestion, but most understandable, I think.

Reply:
Great idea! Actually better then what I suggested